org.openjkdf2.OpenJKDF2.yml

```diff
@@ -0,0 +1,28 @@
+app-id: org.openjkdf2.OpenJKDF2
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+
+command: openjkdf2
+
+finish-args:
+  - --device=all # needed for gamepads
+  - --share=ipc  # needed for X11
+  - --share=network # needed for networking (Multiplayer/Valve GameNetworkingSockets)
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-data:rw # Assets, mods, and saves are located here.
```

Comment:
you don't need this. It's automatically provided (and persisted)

Reply:
Fixed